### DISCUSSION TITLE
Error using data provider filter binding with 'Is In' condition

### DISCUSSION THREAD

**Discussion by applicationuser** [Budibase/budibase#7712] [Help] [answered]
Hello,

Error applying filter 'is in' condition for a data provider with Binding option. When i click on Bolt icon error in console and cannot continue.  need to refresh the page. 

We want to filter data provider based on user selected options from a multi select control 

Budibase cloud
app version: 1.3.11
table source : budibase 

Console error screenshot : 
![image](https://user-images.githubusercontent.com/42922833/189411229-ccb4dc11-e831-4b12-b5b9-7a147aace871.png)

let me know you need any other info

Thank You

**Comment by melohagan** [comment]
Hi @applicationuser 

Could you provide detailed reproduction steps please?
If you would like to share your app export to support@budibase.com I can take a look at that as well (let me know which screen to look at)

**Reply by applicationuser** [reply]
Hello @melohagan 

Unfortunately we cannot share the export. 

Here is the screen setup

Form
   Multi Select Picker with custom options - Status ( In Progress , Completed, Not Started)
   Data Provider with Budibase Table which has Status column 
      Table component to display data based on the data provider filter

workflow we want build is user can filter data by selecting multiple options from picker, and apply that options as filter to Data provider.
On the data provider filters we select status as field to compare with operator as 'Is In' and Binding option.
On clicking bolt icon to provide binding parameter , notice error in console and screen is unresponsive.

let me know if you need further details

![image](https://user-images.githubusercontent.com/42922833/191117962-f2ad4bbc-ffde-4dfd-ac8a-8a86427ed91e.png)

**Comment by melohagan** [comment, marked as answer]
Thanks @applicationuser 

I was able to recreate the bug which I raised here: https://github.com/Budibase/budibase/issues/7878

I'll take a look at this now.
